client/api/utils/index.js

```diff
@@ -0,0 +1,16 @@
+//----------------------------------------------------------------------------------------------------------------------
+// Utilities
+//----------------------------------------------------------------------------------------------------------------------
+
+function deepClone(item)
+{
+    return JSON.parse(JSON.stringify(item));
```

Comment:
I understand why this exists, but I get to hate that it has to, right?

Reply:
Yes, you get to hate it just as much as I do.